ietf-layer0-types-ext.yang

```diff
@@ -51,4 +51,397 @@ module ietf-layer0-types-ext {
+//ietf-layer0-types module
+  typedef standard-mode {
+    type string;
+    description
+      "ITU-T G.698.2 standard mode that guarantees 
+       interoperability.
+       It must be an string with the following format:
+       B-DScW-ytz(v) where all these attributes 
+       are conformant
+       to the ITU-T recomendation";
+    reference "ITU-T G.698.2 (11/2018)";
+  }
+
+  typedef frequency-thz {
+      type decimal64 {
+        fraction-digits 6;
+      }
+      units THz;
+      description
+        "The DWDM frequency in THz, e.g., 193.112500";
+      reference
+        "RFC6205: Generalized Labels for
+        Lambda-Switch-Capable (LSC) Label Switching Routers";
+  }
+  typedef frequency-ghz {
+      type decimal64 {
+        fraction-digits 3;
+      }
+      units GHz;
+      description
+        "The DWDM frequency in GHz, e.g., 193112.500";
+      reference
+        "RFC6205: Generalized Labels for
+        Lambda-Switch-Capable (LSC) Label Switching Routers";
+  }    
+
+// GROUPINGS  
+
+
+ /* supported inverse multiplexing capabilities such as max. OTSiG:OTSi cardinality 
+ It is a transponder attribute not tranceiver*/
+
+/*    leaf multiplexing-cap {
+      type uint32;
+    config false;
+      description "supported inverse multiplexing capabilities
+       such as max. OTSiG:OTSi cardinality";
+    }  */
+
+  
+
+
+/* This grouping represent the list of attributes related to */
+/* optical impairment limits for each operational mode */
+/*(min OSNR, max PMD, max CD, max PDL, Q-factor limit, etc.)*/
+/* It can be utilized also in case of "explicit-mode" */
+
+  grouping common-tranceiver-capabilities-organizational-mode {
+    description "Attributes capabilities related to operational 
+    modes of an optical tranceiver";
+
+
+    leaf line-coding-bitrate {
+      type identyref {
+        base line-coding;
+      }
+      config false;
+      description "Bit rate/line coding of optical tributary signal"
+      reference 
+        "ITU-T G.698.2 section 7.1.2"
+    }
+    leaf max-polarization-mode-dispersion {
+      type decimal64 {
+      fraction-digits 2;
+      range "0..max";
+      }
+      units "ps";
+      config false;
+      description "Maximum acceptable accumulated polarization mode dispersion
+        on the receiver";
+    }
+    leaf max-chromatic-dispersion {
+      type decimal64 {
+      fraction-digits 2;
+      range "0..max";
+      }
+      units "ps/nm";
+      config false;
+      description "Maximum acceptable accumulated chromatic dispersion
+        on the receiver";
+    }
+    leaf max-diff-group-delay  {
+           type int32;
+           config false;
+           description "Maximum Differential group delay of this mode
+                        for this lane";
+    }
+    leaf max-polarization-dependent-loss {
+           type decimal64 {
+             fraction-digits 2;
+           }
+           units dB ;
+           description "Polarization dependent loss";

```

Comment:
Shouldn'tte description be: "Maximum acceptable accumulate polarization dependent loss" ?

Reply:
yes. I will update accordingly